docs/TextField/README.md

```diff
@@ -28,28 +28,83 @@ BpkTextField(context).apply {
 }
 ```
 
+To add a label and/or a helper or error message you can use the `BpkTextInputLayout`:
+
+Example of a text layout with label, error and helper text:
+
+```xml
+<net.skyscanner.backpack.text.BpkTextInputLayout
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:label="Label"
+  app:textInputHelperText="Helper text"
+  app:textInputError="Error message">
+
+  <net.skyscanner.backpack.text.BpkTextField
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:hint="Placeholder"
+    android:text="Content"
+    app:textFieldIconStart="@drawable/bpk_search" />
+</net.skyscanner.backpack.text.BpkTextInputLayout>
+```
+
+Example of a text layout with label, error and helper text:
+
+```Kotlin
+import net.skyscanner.backpack.text.BpkTextField
+import net.skyscanner.backpack.text.BpkTextInputLayout
+
+BpkTextInputLayout(testContext).apply {
+  label = "Label"
+  error = "Error"
+  helperText = "Helper text"
+
+  val textField = BpkTextField(testContext).apply {
+    hint = "Hint"
+    setText("Text")
+  }
+
+  addView(textField)
+}
+```
+
+**Note:** If both error message and helper text are supplied the error message takes precedence.
 
 ## Theme Props
 
+### `BpkTextField`
```

Comment:
I would add a link to new component to root README.md with link to this anchor

Reply:
I was thinking about that but that'll be difficult for the site to refer to both readmes - the web component is already supporting labels etc, so we can't create a separate component on the site for it